### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by piotr-iohk**
### Comments

Various doc updates. Getting rid of references to old testnet and general polishing.

### CODE REVIEW

File: docs/user-guide/common-use-cases/assets.md [hunk 1]
```diff
@@ -271,7 +276,7 @@ $ curl -X POST http://localhost:8090/v2/wallets/73d38c71e4b8b5d71769622ab4f5bfde
 ```
 That's it! I should now receive CBOR-encoded `transaction`, `fee` and `coin_selection` details in response.
 I can now **sign** and **submit** such a transaction just like in [[how-to-make-a-transaction]].
-Once submitted my freshly minted NFT should be added to my wallet balance!
+Once transaction submitted your freshly minted NFT should be added to the wallet balance!
```
Comment (paweljakubas) [inline docs/user-guide/common-use-cases/assets.md]:
```suggestion
Once the transaction is submitted your freshly minted NFT should be added to the wallet balance!
```

File: docs/user-guide/common-use-cases/delegation.md [hunk 2]
```diff
@@ -135,6 +135,7 @@ $ curl -X POST http://localhost:8090/v2/wallets/1b0aa24994b4181e79116c131510f2ab
 -H "Content-Type: application/json"
 ```
 
-> :information_source: Note that all rewards will be withdrawn only if you add {"withdrawal":"self"} to the payload. You can do it in a single transaction though as above.
+After constructing delegation quit transaction I should receive CBOR-encoded `transaction`, `fee` and `coin_selection` details in response.
```
Comment (paweljakubas) [inline docs/user-guide/common-use-cases/delegation.md]:
```suggestion
After constructing a quitting-from-pool transaction I should receive CBOR-encoded `transaction`, `fee` and `coin_selection` details in the corresponding response.
```

File: docs/user-guide/common-use-cases/delegation.md [hunk 3]
```diff
@@ -135,6 +135,7 @@ $ curl -X POST http://localhost:8090/v2/wallets/1b0aa24994b4181e79116c131510f2ab
...
+I can now **sign** and **submit** such a transaction just like in [[how-to-make-a-transaction]].
 
-> :information_source: Refer to [[how-to-make-a-transaction]] for details on signing and submitting it to the network.
+> :information_source: Note that all rewards will be withdrawn only if you add {"withdrawal":"self"} to the payload. You can do it in a single transaction though as above.
```
Comment (paweljakubas) [inline docs/user-guide/common-use-cases/delegation.md]:
```suggestion
> :information_source: Note that all rewards will be withdrawn only if you add {"withdrawal":"self"} to the payload. You can realize this within a single transaction as shown above.
```

**Review by paweljakubas — APPROVED** [review]

File: docs/user-guide/common-use-cases/delegation.md (hunk 2 above)
Comment (jonathanknowles) [reply docs/user-guide/common-use-cases/delegation.md]: > "After constructing a transaction that undelegates from a stake pool, I should receive a CBOR-encoded `transaction`, `fee`, and `coin_selection` in the response."

File: docs/user-guide/common-use-cases/assets.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ title: Managing native assets
 
  - [[how-to-start-wallet-server]]
  - [[how-to-create-a-wallet]]
- - In order to be able to send transactions we need funds on the wallet. In case of [Testnet](https://testnets.cardano.org/en/testnets/cardano/overview/) we can request tADA from the [faucet](https://testnets.cardano.org/en/testnets/cardano/tools/faucet/).
+ - In order to be able to send transactions we need funds on the wallet. In case of `preview` and `preprod` [testnets](https://testnets.cardano.org/en/testnets/cardano/overview/) we can request tADA from the [faucet](https://testnets.cardano.org/en/testnets/cardano/tools/faucet/).
```
Comment (jonathanknowles) [inline docs/user-guide/common-use-cases/assets.md]: > "In order to be able to send transactions, our wallet must have funds."

Or

> "In order to be able to send transactions, we must have a wallet with funds."

File: docs/user-guide/common-use-cases/assets.md [hunk 5]
```diff
@@ -62,18 +62,23 @@ You can also list assets that were ever associated with the wallet.
 
 ## Assets off-chain metadata
 
-Issuers of native assets may put some metadata about them in the off-chain metadata server, like **Cardano Token Registry**.
+Issuers of native assets may put some off-chain metadata about them in the **Cardano Token Registry**. There are separate instances of token registry for `mainnet` and testnets (e.g. `preview` or `preprod`). Users can submit metadata to the repository which will be then served via corresponding server.
```
Comment (jonathanknowles) [inline docs/user-guide/common-use-cases/assets.md]: Perhaps:
> Issuers of native assets may submit off-chain metadata relating to those assets to the **Cardano Token Registry**. There are separate instances of the token registry for `mainnet` and for each test network (e.g. `preview` or `preprod`). Metadata submitted to the registry will be then served via the corresponding server.

File: docs/user-guide/common-use-cases/assets.md [hunk 6]
```diff
@@ -62,18 +62,23 @@ You can also list assets that were ever associated with the wallet.
...
+
+|  | Mainnet | Testnets  |
+|--|--|--|
+| **Repository** | https://github.com/cardano-foundation/cardano-token-registry | https://github.com/input-output-hk/metadata-registry-testnet |
+| **Server** | https://tokens.cardano.org | https://metadata.cardano-testnet.iohkdev.io |
 
 > :information_source: Read more: [Cardano Token Registry](https://developers.cardano.org/docs/native-tokens/token-registry/cardano-token-registry), [CIP26](https://cips.cardano.org/cips/cip26).
 
-Cardano-wallet is capable of reading that metadata and serving it along while listing assets. All you have to do is to start wallet with `--token-metadata-server` parameter pointing to an off-chain metadata server.
+Cardano-wallet is capable of reading that metadata and serving it along while listing assets. All you have to do is to start wallet with `--token-metadata-server` parameter pointing to an off-chain metadata server corresponding to your network.
```
Comment (jonathanknowles) [inline docs/user-guide/common-use-cases/assets.md]: > Cardano-wallet is capable of reading that metadata and serving it when listing assets. All you have to do is to start the wallet with the `--token-metadata-server` option, specifying an off-chain metadata server that corresponds to your network.

File: docs/user-guide/common-use-cases/assets.md [hunk 7]
```diff
@@ -142,7 +147,7 @@ $ curl -X POST http://localhost:8091/v2/wallets/73d38c71e4b8b5d71769622ab4f5bfde
 "policy_vk12d0gdel9u6px8wf3uv4z6m4h447n9qsad24gztaku8dzzdqfajzqfm3rr0"
 ```
 
-Once we sort it out we are all set and we can proceed into minting an NFT from our wallet!
+Once we sort it out we are all set and we can proceed into minting NFT from our wallet!
```
Comment (jonathanknowles) [inline docs/user-guide/common-use-cases/assets.md]: > Once we have finished, we can proceed to minting NFTs from our wallet!

File: docs/user-guide/common-use-cases/assets.md (hunk 1 above)
Comment (jonathanknowles) [inline docs/user-guide/common-use-cases/assets.md]: > After the transaction has been submitted, your freshly-minted NFT should appear in your wallet balance!

File: docs/user-guide/common-use-cases/delegation.md (hunk 3 above)
Comment (jonathanknowles) [reply docs/user-guide/common-use-cases/delegation.md]: > You can achieve this with a single transaction, as shown above.

File: docs/user-guide/common-use-cases/how-to-start-wallet-server.md [hunk 8]
```diff
@@ -10,33 +10,49 @@ The easiest and most common way of managing your funds on the Cardano blockchain
...
-  --config testnet-config.json \
-  --topology testnet-topology.json \
+  --config config.json \
+  --topology topology.json \
   --database-path ./db \
   --socket-path /path/to/node.socket
 ```
 2. Start wallet:
+
+When starting wallet against any testnet environment like `preview` or `preprod` we need to feed wallet with `byron-genesis.json` file:
```
Comment (jonathanknowles) [inline docs/user-guide/common-use-cases/how-to-start-wallet-server.md]: > When starting a wallet instance that targets a testing environment such as `preview` or `preprod`, we need to provide a `byron-genesis.json` file to the wallet:

File: docs/user-guide/common-use-cases/how-to-start-wallet-server.md [hunk 9]
```diff
@@ -10,33 +10,49 @@ The easiest and most common way of managing your funds on the Cardano blockchain
...
 ```bash
 $ cardano-wallet serve --port 8090 \
   --node-socket /path/to/node.socket \
-  --testnet testnet-byron-genesis.json \
+  --mainnet \
   --database ./wallet-db \
-  --token-metadata-server https://metadata.cardano-testnet.iohkdev.io/
+  --token-metadata-server https://tokens.cardano.org
 ```
+> :information_source: Notice that we use different urls for `mainnet` and `testnet` `--token-metadata-server` option. These are links to [Cardano Token Registry](https://developers.cardano.org/docs/native-tokens/token-registry/cardano-token-registry) servers. See [[assets]] for more information.
```
Comment (jonathanknowles) [inline docs/user-guide/common-use-cases/how-to-start-wallet-server.md]: > Notice that we use different URLs for `mainnet` and test networks with the `--token-metadata-server` option. These URLs point to [Cardano Token Registry](https://developers.cardano.org/docs/native-tokens/token-registry/cardano-token-registry) servers. See [[assets]] for more information.

File: docs/user-guide/common-use-cases/assets.md (hunk 1 above)
Comment (jonathanknowles) [reply docs/user-guide/common-use-cases/assets.md]: > After the transaction has been submitted, your freshly-minted NFT should appear in your wallet balance!

**Review by jonathanknowles — COMMENTED** [review]
Hi @piotr-iohk 

Just some suggested rewordings!

**Pull request merged**